src/lodepng/lodepng.cpp

```diff
@@ -228,7 +228,7 @@ static char* alloc_string(const char* in) {
 #if defined(LODEPNG_COMPILE_DECODER) || defined(LODEPNG_COMPILE_PNG)
 static unsigned lodepng_read32bitInt(const unsigned char* buffer) {
   return (((unsigned)buffer[0] << 24u) | ((unsigned)buffer[1] << 16u) |
-         ((unsigned)buffer[2] << 8u) | (unsigned)buffer[3]);
```

Comment:
This (and a few other lines) are how it is formatted right now in mainline lodepng, so I will leave it as so that this lodepng does not differ more from it just based on formatting.